src/pages/Lend/Filter/FilterSectionSort.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+	<filter-menu-section :title="title" :open="open" :result-count="resultCount">
+		<ais-sort-by :items="defaultSortIndices">
```

Comment:
Please pass defaultSortIndices down into this component. This so we don't have to include configure or the mixin here.

Alternatively, you could pull the fundraisingIndices + defaultSortIndices() out of algolia-config-mixin into a 2nd algolia-sort-indices-mixin.

Reply:
@mcstover PR updated.